Mathlib/CategoryTheory/Sites/Hypercover/Zero.lean

```diff
@@ -384,6 +410,32 @@ def interLift (f : G.Hom E) (g : G.Hom F) :
 
 end
 
+/-- If `{Uᵢ}` covers `X`, the pre-`0`-hypercover `{Uᵢ ×[Z] Y}` of `X ×[Z] Y` is isomorphic
+to the pullback of `{Uᵢ}` along the first projection. -/
+noncomputable
+def pullbackCoverOfLeftIsoPullback₁ {X Y Z : C} (f : X ⟶ Z) (g : Y ⟶ Z) [HasPullback f g]
+    (E : PreZeroHypercover X) [∀ i, HasPullback (pullback.fst f g) (E.f i)]
+    [∀ i, HasPullback (E.f i) (pullback.fst f g)] :
+    E.pullbackCoverOfLeft f g ≅ pullback₁ (pullback.fst f g) E := by
+  refine PreZeroHypercover.isoMk (.refl _) ?_ ?_
+  · intro i
+    exact (pullbackRightPullbackFstIso _ _ _).symm ≪≫ pullbackSymmetry _ _
+  · intro i
+    apply pullback.hom_ext <;> simp
```

Comment:
Automation works:
```suggestion
    E.pullbackCoverOfLeft f g ≅ pullback₁ (pullback.fst f g) E :=
  PreZeroHypercover.isoMk (.refl _)
    (fun _ ↦ (pullbackRightPullbackFstIso _ _ _).symm ≪≫ pullbackSymmetry _ _)
```
(And similarly for the one below.)
Could you also change the order of the parameters for the isomorphisms similarly as for the definitions of the `0`-prehypercovers?